image.yaml

```diff
@@ -2,22 +2,22 @@ schema_version: 1
 
 name: "redhat-sso-7/sso71"
 description: "Red Hat Single Sign-On 7.1 container image"
-version: "7.1.2"
+version: "7.1.3"
 from: "jboss-eap-7/eap70:latest"
 envs:
     - name: "JBOSS_PRODUCT"
       value: "sso"
     - name: "JBOSS_SSO_VERSION"
-      value: "7.1.2"
+      value: "7.1.3"
     - name: "PRODUCT_VERSION"
-      value: "7.1.2"
+      value: "7.1.3"
 modules:
       install:
           - name: sso
 artifacts:
       # keycloak-server-overlay-2.5.10.Final-redhat-1.zip
```

Comment:
This comment probably should be updated too, or removed if note required. But I would probably leave it since the `path` is not showing what build it is.

Reply:
I pushed a replacement commit that updates this comment too.